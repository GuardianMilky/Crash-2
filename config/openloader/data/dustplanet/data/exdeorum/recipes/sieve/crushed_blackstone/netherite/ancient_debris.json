{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:crushed_blackstone"},"mesh":"exdeorum:netherite_mesh","result":"minecraft:ancient_debris","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.1}}