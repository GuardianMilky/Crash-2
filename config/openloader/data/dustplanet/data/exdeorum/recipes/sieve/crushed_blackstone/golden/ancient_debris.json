{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:crushed_blackstone"},"mesh":"exdeorum:golden_mesh","result":"minecraft:ancient_debris","result_amount":{"type":"minecraft:binomial","n":1.0,"p":0.05}}